feature-tests/utils/ais-events.ts

```diff
@@ -40,12 +40,12 @@ export const aisEvents: {
     timestamp: seconds,
     event_timestamp_ms: ms,
     event_name: 'TICF_ACCOUNT_INTERVENTION',
-    component_id: 'TICF_CRI',
```

Comment:
minor point, but with a "real" event, I do not think that this value would vary between different intervention types.. so I think it is a bit confusing to change this way

Reply:
Hi @chiarapascucci-gds, updated the files